### DISCUSSION TITLE
包含div的img无法正常显示

### DISCUSSION THREAD

**Discussion by zxzf1234** [wangeditor-next/wangEditor-next#370] [Q&A] [answered]
## bug 描述

`<div><img src="http://gips3.baidu.com/it/u=1022347589,1106887837&fm=3028&app=3028&f=JPEG&fmt=auto?w=960&h=1280" border="0"></div>`
这段html无法显示，

## 你预期的样子是？

可以正常显示

## 系统和浏览器及版本号

- win11
- chrome 130.0.6723.119

## wangEditor-next 版本

5.6.14

## demo 能否复现该 bug ？

demo不能复现，沙盒可以复现

- 沙盒链接:https://stackblitz.com/edit/vue3-wangeditor-demo-nf8grm?file=src%2Fcomponents%2FBasicEditor.vue
![image](https://github.com/user-attachments/assets/220ac5a9-82b0-4c53-b76e-0b5f7f4f4684)

本地使用wangEditor没有这个问题
使用wangEditor的沙盒也有问题

**Comment by wjw020206** [comment, marked as answer]
wangEditor-next 只识别 `editor.getHtml()` 生成的 html 格式，不可以随意自定义 html 代码
建议使用 `<p><img src="http://gips3.baidu.com/it/u=1022347589,1106887837&fm=3028&app=3028&f=JPEG&fmt=auto?w=960&h=1280" border="0"></p>` 

你用的 demo 应该是这个吧 https://cycleccc.github.io/demo/set-html.html ，这个 demo 可以是因为 setHtml 按钮调用了 `dangerouslyInsertHtml` 方法，会对 `div` 进行过滤

**Comment by cycleccc** [comment]
已明确仅支持编辑器生成的 HTML 结构，使用 <p><img ...></p> 可处理该场景。本讨论现关闭。
